src/js/rebuses.js

```diff
@@ -887,6 +887,11 @@ const rebuses = [
     symbols: ['🍯', '+', '🐝'],
     words: ['honeybee'],
     hint: ['An insect that makes a sweet treat.']
+  },
+  {
+    symbols: [ '😢', '+',  '👶'],
```

Comment:
I think the Travis build failure is saying that you've got an extra space right after `[` and another right before `'👶'`. If you remove those, the failure will go away and you'll be good to go!